test/runner/e2e_test.cpp

```diff
@@ -56,6 +58,58 @@ class EndToEndTest : public DBTest {
     }
 };
 
+class MultipleConnTest : public MultipleDBTest {
+public:
+    explicit MultipleConnTest(TestGroup::DatasetType datasetType, std::string dataset,
+        uint64_t bufferPoolSize, uint64_t checkpointWaitTimeout,
+        std::vector<std::unique_ptr<TestStatement>> testStatements)
+        : datasetType{datasetType}, dataset{std::move(dataset)}, bufferPoolSize{bufferPoolSize},
+          checkpointWaitTimeout{checkpointWaitTimeout}, testStatements{std::move(testStatements)} {}
+
+    void SetUp() override {
+        setUpDataset();
+        BaseGraphTest::SetUp();
+        systemConfig->bufferPoolSize = bufferPoolSize;
+        createDB(checkpointWaitTimeout);
+        if (dataset != "empty") {
+            init_graph = true;
+        }
+    }
+
+    void setUpDataset() {
+        parquetTempDatasetPath = generateParquetTempDatasetPath();
+        dataset = TestHelper::appendKuzuRootPath("dataset/" + dataset);
+        if (datasetType == TestGroup::DatasetType::CSV_TO_PARQUET) {
+            dataset =
+                CSVToParquetConverter::convertCSVDatasetToParquet(dataset, parquetTempDatasetPath);
+        }
+    }
+
+    void TearDown() override {
+        FileUtils::removeDir(databasePath);
+        FileUtils::removeDir(parquetTempDatasetPath);
+    }
+
+    void TestBody() override { runTest(testStatements, init_graph); }
+    std::string getInputDir() override { return dataset + "/"; }
+
+private:
+    TestGroup::DatasetType datasetType;
+    std::string dataset;
+    std::string parquetTempDatasetPath;
+    uint64_t bufferPoolSize;
+    uint64_t checkpointWaitTimeout;
+    std::vector<std::unique_ptr<TestStatement>> testStatements;
+    bool init_graph;
```

Comment:
Let's be consistent and use camel case, so initGraph

Reply:
ok